enterprise/server/test/integration/remote_cache/remote_cache_test.go

```diff
@@ -190,3 +197,93 @@ func TestBuild_RemoteCacheFlags_Compression_SecondBuildIsCached(t *testing.T) {
 		"second build should be cached since anonymous users have cache write capabilities by default",
 	)
 }
+
+func TestBuild_RemoteCache_ScoreCard(t *testing.T) {
+	app := buildbuddy_enterprise.RunWithConfig(
+		t, buildbuddy_enterprise.NoAuthConfig,
+		"--redis_command_buffer_flush_period=0",
+		"--cache_stats_finalization_delay=0")
+	bbService := app.BuildBuddyServiceClient(t)
+	ctx := context.Background()
+	ws := testbazel.MakeTempWorkspace(t, workspaceContents)
+	iid := newUUID(t)
+	buildFlags := []string{"//:hello.txt", "--invocation_id=" + iid}
+	buildFlags = append(buildFlags, app.BESBazelFlags()...)
+	buildFlags = append(buildFlags, app.RemoteCacheBazelFlags()...)
+
+	{
+		result := testbazel.Invoke(ctx, t, ws, "build", buildFlags...)
+
+		assert.NoError(t, result.Error)
+		inv := getInvocationWithStats(t, ctx, bbService, iid)
+		sc := inv.GetScoreCard()
+		clearActionIDs(sc)
+		expectedSC := &capb.ScoreCard{
+			Misses: []*capb.ScoreCard_Result{
+				{
+					ActionMnemonic: "Genrule",
+					TargetId:       "//:hello_txt",
+				},
+			},
+		}
```

Comment:
this is fine here but probably doesn't "scale" as well as a custom [partial matcher](https://stackoverflow.com/questions/68349850/how-to-assert-a-partial-match-with-stretchr-testify-mock-assertcalled) would.

(fine for now, just making sure you have options)